cmd/api/src/database/migration/agi.go

```diff
@@ -42,7 +42,7 @@ func SelectorToObjectID(rawSelector string) string {
 }
 
 func (s *Migrator) updateAssetGroups() error {
-	return s.db.Transaction(func(tx *gorm.DB) error {
+	return s.Db.Transaction(func(tx *gorm.DB) error {
```

Comment:
Our usual standard for exporting the name `db` is `DB`, which follows the naming style used in other libraries